FeedStoreChallenge/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Joshua Bryson on 9/9/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+final class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+
+	@NSManaged var cache: ManagedCache
+}
+
+extension ManagedFeedImage {
+	class func images(from images: [LocalFeedImage], in context: NSManagedObjectContext, with cache: ManagedCache) -> NSOrderedSet {
```

Comment:
Suggestion:
Since the class is private, all `class` methods work the same way as `static` ones. The main difference between `static` and `class` (from architectural perspective) is that `class` method can be overridden whilst `static` can't.
Using `class` methods in such a case can be misleading thus it is a little bit more correct to use `static`
```suggestion
	static func images(from images: [LocalFeedImage], in context: NSManagedObjectContext, with cache: ManagedCache) -> NSOrderedSet {
```